game-cards/SGgame.js

```diff
@@ -501,7 +501,7 @@ class SGgame extends GameConsole {
 			output += '<div style="display: inline-block; float: right; width: 50%; height: 100%; text-align: center;"><div class="movemenu"><center>';
 			for (let m = 0; m < pokemon.moves.length; m++) {
 				move2 = Dex.getMove(pokemon.moves[m]);
-				let maxpp2 = move.noPPBoosts ? move2.pp : move2.pp * 8 / 5;
+				let maxpp2 = pokemon.moves[m].noPPBoosts ? move2.pp : move2.pp * 8 / 5;
```

Comment:
`pokemon.moves[m]` is a string, the issue here seems to be that the variable is `move2` and we were calling `move`, changing it to `move2` should fix it.

At some point i need to declare these variables outside of the switch statement to make it all cleaner.